polkadot/node/subsystem-bench/examples/estimate_cpu_requirements.yaml

```diff
@@ -0,0 +1,47 @@
+## Estimate CPU requirements on a network with 500 validators 100 cores
+## and 5 backing validators per core.
+TestConfiguration:
+# Approval voting best case scenario
+- objective: !ApprovalVoting
+    coalesce_mean: 3.0
+    coalesce_std_dev: 1.0
+    enable_assignments_v2: true
+    last_considered_tranche: 89
+    stop_when_approved: true
+    coalesce_tranche_diff: 12
+    num_no_shows_per_candidate: 0
+    workdir_prefix: "/tmp/"
+  n_validators: 500
+  n_cores: 100
+  min_pov_size: 1120
+  max_pov_size: 5120
+  peer_bandwidth: 524288000000
+  bandwidth: 524288000000
+  num_blocks: 3
+  connectivity: 100
+  latency: null
+
+- objective: DataAvailabilityWrite
+  n_validators: 500
+  n_cores: 100
+  max_validators_per_core: 5
```

Comment:
Is this backing?

You won't have much of a CPU requirement estimate without approvals, but maybe they're set elsewhere?

Reply:
This is the one measuring the approval voting CPU: https://github.com/paritytech/polkadot-sdk/pull/4323/files#diff-3173d0f01ac97698eec2cfc0da96349100b8cdfefa58f42369e142b39f0997b7R5.

Reply:
I meant the `needed_approvals` setting.

Reply:
Yes, that a configuration parameters as well, when it is not present the default version is used, which is 30 as all of our production chains.

https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/node/subsystem-bench/src/lib/configuration.rs#L75